src/test/java/com/bobocode/svydovets/ioc/core/autowire/AutowiredMethodAnnotationBeanPostProcessorTest.java

```diff
@@ -12,25 +12,13 @@
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-public class AutowiredAnnotationBeanPostProcessorTest {
+public class AutowiredMethodAnnotationBeanPostProcessorTest {
 
     private AnnotationConfigApplicationContext context;
 
     @BeforeEach
     public void setUp() {
-        context = new AnnotationConfigApplicationContext(AutowirePackageBeansConfig.class);
-    }
-
-    @Test
-    void shouldNotNullWhenFieldIsInjected() {
-        EditService editService = context.getBean(EditService.class);
-        assertThat(editService.getMessageService()).isNotNull();
-    }
-
-    @Test
-    void shouldReturnNullIfFieldNotInjected() {
-        EditService editService = context.getBean(EditService.class);
-        assertThat(editService.getCommonService()).isNull();
+        context = new AnnotationConfigApplicationContext(AutowireByMethodPackageBeansConfig.class);
```

Comment:
Please, add test with **new service** that has 2 fields and **2 separated setters** with @Autowired annotation to verify the values will be injected